### DISCUSSION TITLE
Is the version of waitress that MLFlow depends on listed anywhere?

### DISCUSSION THREAD

**Discussion by cranialsurge** [mlflow/mlflow#5757] [Q&A] [answered]
Hi, we consume MLFlow via Azure Databricks. It seems to bring in waitress as a dependency. The current version of waitress being brought in is 2.0.0. However there was a recently found security vulnerability in waitress that is called out here: https://cve.mitre.org/cgi-bin/cvename.cgi?name=2022-24761.

This has been addressed in waitress v2.1.1.  Assuming a future version of MLFlow will(or may already have been) be updated to use the new waitress versions. However I don't see waitress listed in any of the requirement files in this repo. to check what version is being used. Couldn't find it in release notes either. Any idea if this information is available anywhere?

**Comment by harupy** [comment, marked as answer]
`waitress` is listed here:

https://github.com/mlflow/mlflow/blob/c577c7f199d9efa076344785dabb2121edb7e6c8/setup.py#L83

It's unpinned so the latest version is used unless other packages pin it.
